### PR TITLE
Create content for scourge under the staking economics track

### PR DESCRIPTION
**Pull request by gconnect**
### What the PR Does
- Fix change lowercase to upcase in titles
- format the reference section
- create content for scourge under the staking economics track

### CODE REVIEW

File: docs/wiki/research/roadmap.md [hunk 1]
```diff
@@ -74,74 +74,100 @@ Upgrades related to censorship resistance, decentralization and mitigating proto
...
 Other upgrades that don't fit well into the previous categories.
 
 ## Resources
 
-[^1]: EIP-2982: Serenity Phase 0 https://eips.ethereum.org/EIPS/eip-2982, [[archived]](https://web.archive.org/web/20230928204358/https://eips.ethereum.org/EIPS/eip-2982)
+[^1]: [EIP-2982: Serenity Phase 0](https://eips.ethereum.org/EIPS/eip-2982), [[archived]](https://web.archive.org/web/20230928204358/https://eips.ethereum.org/EIPS/eip-2982)
+
+[^2]: [EIP-4895: Beacon chain push withdrawals](https://eips.ethereum.org/EIPS/eip-4895), [[archived]](https://web.archive.org/web/20240415201815/https://eips.ethereum.org/EIPS/eip-4895)
+
+[^4]: [VB's SSF notes](https://notes.ethereum.org/@vbuterin/single_slot_finality), [[archived]](https://web.archive.org/web/20240330010706/https://notes.ethereum.org/@vbuterin/single_slot_finality)
```
Comment (raxhvl) [inline docs/wiki/research/roadmap.md:108]: Hi @gconnect thanks for improving this.

I noticed that the foot note is not enumerated correctly. no. 3 is missing.

**Review by raxhvl — CHANGES_REQUESTED** [review]

File: docs/wiki/research/roadmap.md [hunk 2]
```diff
@@ -74,74 +74,100 @@ Upgrades related to censorship resistance, decentralization and mitigating proto
...
 | Preconfirmations                   | MEV-Track |                 -                 |                       Users preconfirmations on transaction execution, for a competitive user experience in Ethereum interactions                        |                                                                                                                                  Block builders could publicly agree to include transactions with a priority fee over a certain amount, and send users a receipt indicating their intent to include the transaction in a specific block                                                                                                                                  | [in research](/wiki/research/Preconfirmations/Preconfirmations.md)[^27]                                    |
+      
+Here is a summarized table of the provided content for the Staking Economics/Experiences Track:
```
Comment (taxmeifyoucan) [inline docs/wiki/research/roadmap.md:78]: I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

Comment (gconnect) [reply docs/wiki/research/roadmap.md]: @raxhvl and @taxmeifyoucan I just made the requested changes now. please let me know if its okay.

**Comment by gorondan** [comment]
Thank you for your contribution to the Roadmap page, @gconnect. I'll review ASAP, busy week

**Comment by gconnect** [comment]
> Thank you for your contribution to the Roadmap page, @gconnect. I'll review ASAP, busy week

Alright thanks, I will be looking forward to your review!

**Review by gorondan — APPROVED** [review]

File: docs/wiki/research/roadmap.md (hunk 2 above)
Comment (gorondan) [reply docs/wiki/research/roadmap.md]: > I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree

> I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree with @taxmeifyoucan here, @gconnect. MAX_EB line entry is OK, the other ones should follow that and the above formatting: first column should be the proposed roadmap Upgrade (e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you passed in the first column, actually)

Comment (gorondan) [reply docs/wiki/research/roadmap.md]: > I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree with @taxmeifyoucan here, @gconnect. MAX_EB line entry is OK, the other ones should follow that and the above formatting: first column should be the proposed roadmap Upgrade (e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you passed in the first column, actually)

Comment (gconnect) [reply docs/wiki/research/roadmap.md]: > > I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics
> 
> I agree with @taxmeifyoucan, @gconnect. MAX_EB line entry is OK, the other ones should follow that, and the above formatting: first column should be the proposed roadmap Upgrade itself(e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you actually passed in the first column)

I just made some adjustment. Please take a look and let me know what you think

**Pull request merged**